### ISSUE TITLE
api.HTMLElement.dragover_event - Missing coordinates

### ISSUE THREAD

**Issue by stefnotch** [mdn/browser-compat-data#21784]
### What type of issue is this?

Browser bug (a bug with a feature that may impact site compatibility)

### What information was incorrect, unhelpful, or incomplete?

A few of the drag events, such as https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragover_event suffer from
https://bugzilla.mozilla.org/show_bug.cgi?id=505521

Basically, the mouse coordinates are missing. 

### What browsers does this problem apply to, if applicable?

Firefox

### What did you expect to see?

I expected the compatibility data to reflect on this.

(Ideally, I'd expect the correct people at Firefox to be notified about the bug, so that they can finally fix it. I'll happily donate 50 bucks or whatever is needed to make this happen.)

### Did you test this? If so, how?

-

### Can you link to any release notes, bugs, pull requests, or MDN pages related to this?

https://bugzilla.mozilla.org/show_bug.cgi?id=505521

I actually reported this in the past, but somehow the info from https://github.com/mdn/browser-compat-data/pull/4961 seems to have gotten lost

### MDN URL

https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/dragover_event

### MDN metadata

<details>
<summary>MDN page report details</summary>

* Query: `api.HTMLElement.dragover_event`
* Report started: 2023-12-30T21:23:03.354Z

</details>

**Comment by caugner** [comment]
I don't think this implies partial implementation, but we can definitely add a note, referencing [bug 505521](https://bugzilla.mozilla.org/show_bug.cgi?id=505521).

The question is whether where to add this note:

- on the `drag*` events, or
- on the `DragEvent` interface, or
- on the `MouseEvent.{client,page,screen}{X,Y}` properties

I added this issue to the meeting agenda for discussion.

**Comment by caugner** [comment]
We have discussed this in today's BCD meeting:

According to [this comment](https://bugzilla.mozilla.org/show_bug.cgi?id=505521#c95), this only affects the `drag` event itself. Therefore we aligned on adding this note to the `drag_event` feature.
